app/src/main/java/co/icanteach/projectx/common/Extensions.kt

```diff
@@ -8,16 +8,19 @@ import androidx.databinding.ViewDataBinding
 import androidx.lifecycle.LifecycleOwner
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.Observer
-import io.reactivex.Observable
-import io.reactivex.disposables.CompositeDisposable
-import io.reactivex.disposables.Disposable
-import io.reactivex.schedulers.Schedulers
+import io.reactivex.rxjava3.core.Observable
+import io.reactivex.rxjava3.disposables.CompositeDisposable
+import io.reactivex.rxjava3.disposables.Disposable
+import io.reactivex.rxjava3.schedulers.Schedulers
 
 
 /**
  * http://kotlinextensions.com/
  */
-fun <T : ViewDataBinding> ViewGroup?.inflate(@LayoutRes layoutId: Int, attachToParent: Boolean = true): T {
```

Comment:
Should we write extension function for `Nullable` types?

There is a bad example about `null`. null.toString() may work as returning `"null"`. 

Idiomaticaly, `null.something` should throw an `NPE` in Java but in Kotlin it should not be compiled at all. 

Also if `ViewGroup` is `null`, this can be classified as `IllegalState` in the app. Because viewGroup in `Adapter<VH>` defined as :

```kt
        public abstract VH onCreateViewHolder(@NonNull ViewGroup parent, int viewType);
```

Final suggestion is just to remove `?` from here